### DISCUSSION TITLE
Correct kernel loaded but notebook won't run

### DISCUSSION THREAD

**Discussion by LFL-Lab** [microsoft/vscode-jupyter#16141] [Questions and Answers]
I am running python code in jupyter notebook in vscode and recently I started experiencing weird issues. Specifically, I cannot run any cells even though I have selected the correct python interpreter. 

**Environment Setup**
Windows
- python==3.9.13
- ipykernel==6.29.5
- jupyter==1.1.1
- jupyterlab==4.2.5
- notebook==7.2.2
**What I have tried**:
- Reinstalling vscode
- Reinstalling jupyter and copilot extensions
- Restarting PC
- Reload kernel
**Symptoms**
- Even simplest cell stuck at execution
- Can't interrupt kernel
- Stuck at reloading kernel

Any  help is much appreciated. Thank you.

**Comment by DonJayamanne** [comment]
> Specifically, I cannot run any cells even though

What do you mean by this? Are you getting errors, messages, or buttons are disabled?

Please could you enable logging as follows:
* Open settings & go into `Jupyter->Logging`
* Change the value to `trace`
* Reload VS Code,
* Attempt to repro the issue & then copy the output from the `Jupyter` output panel (use the command `Jupyter: Show Output` to get to the logs).

**Reply by LFL-Lab** [reply]
I click on the execute button and nothing happens after that. If I try to restart the kernel or interrupt cell execution VScode gets stuck. There are no errors or messages. I don't see any errors in the jupyter output. In addition, this behavior is specific to jupyter notebooks launched in vscode. Everything works fine if open a terminal, run "jupyter notebook" and then open the ipynb file in the browser.

**Reply by DonJayamanne** [reply]
thanks, please can you share the requested logs

**Comment by LFL-Lab** [comment]
`8:30:47.042 [debug] KernelProvider switched kernel to id = .jvsc74a57bd0a255bef7dadeb5597c65f953c10788a4d170ae1aea2c4be952474a3a3fcfa6a1.c:\Users\~\darpa-measurements\darpa-measurements\darpa-meas-env\Scripts\python.exe.c:\Users\~\darpa-measurements\darpa-measurements\darpa-meas-env\Scripts\python.exe.-m#ipykernel_launcher
18:30:47.043 [debug] start the kernel, options.disableUI=true for ~\darpa-measurements\darpa-measurements\darpa_measurements.ipynb
18:30:47.044 [debug] Controller selection change completed
18:30:47.044 [info] Starting Kernel (Python Path: ~\darpa-measurements\darpa-measurements\darpa-meas-env\Scripts\python.exe, Venv, 3.9.13) for '~\darpa-measurements\darpa-measurements\darpa_measurements.ipynb' (disableUI=true)
18:30:47.044 [trace] Creating raw notebook for resource '~\darpa-measurements\darpa-measurements\darpa_measurements.ipynb'
18:30:47.046 [trace] Registering commtarget jupyter.widget
18:30:47.053 [debug] Interpreter for Pylance for Notebook URI "~\darpa-measurements\darpa-measurements\darpa_measurements.ipynb" is ~\darpa-measurements\darpa-measurements\darpa-meas-env\Scripts\python.exe
18:30:47.069 [info] Process Execution: ~\darpa-measurements\darpa-measurements\darpa-meas-env\Scripts\python.exe -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
18:30:47.280 [debug] Launching kernel .jvsc74a57bd0a255bef7dadeb5597c65f953c10788a4d170ae1aea2c4be952474a3a3fcfa6a1.c:\Users\~\darpa-measurements\darpa-measurements\darpa-meas-env\Scripts\python.exe.c:\Users\~\darpa-measurements\darpa-measurements\darpa-meas-env\Scripts\python.exe.-m#ipykernel_launcher for ~\darpa-measurements\darpa-measurements\darpa_measurements.ipynb in ~\darpa-measurements\darpa-measurements with ports 9017, 9016, 9020, 9018, 9019
18:30:47.727 [trace] ipykernel version & path 6.29.5, ~\darpa-measurements\darpa-measurements\darpa-meas-env\lib\site-packages\ipykernel\__init__.py for ~\darpa-measurements\darpa-measurements\darpa-meas-env\Scripts\python.exe
`

**Reply by DonJayamanne** [reply]
Please can you share the full logs when you run into this issue.

**Comment by LFL-Lab** [comment]
[full_logs.jsonc](https://github.com/user-attachments/files/17510955/full_logs.jsonc)
